packages/billing/stripe-webhook-handlers/checkout-session-completed.ts

```diff
@@ -56,6 +58,14 @@ export async function handleCheckoutSessionCompleted(
       stripeCustomerId = user.stripeCustomerId;
     }
 
+    if (stripeCustomerEmail) {
```

Comment:
Isn't email always present?

Reply:
Right, since it is required when filling in customer details - I removed the email check